ProcessMaker/processmaker
# Edit Categories Modal

closes #2534 

## Changes
- Change the editing category using the modal component.
- Remove controller edit category (Process, screen, script)
- remove web api
- remove edit category blade

## Test

- Create a new category (Process, screen, script).
- Edit category, a category editing modal should be displayed.
- save, you must close modal and update only the list.

## Screenshot

![image](https://user-images.githubusercontent.com/1747025/69440997-3820a700-0d20-11ea-84dd-680875e4b0d4.png)
